globalconf.go

```diff
@@ -98,14 +98,14 @@ func (g *GlobalConf) Delete(flagSetName, flagName string) error {
 	return nil
 }
 
-// Parses the config file for the provided flag set.
+// ParseSet parses the config file for the provided flag set.
 // If the flags are already set, values are overwritten
 // by the values in the config file. Defaults are not set
 // if the flag is not in the file.
 func (g *GlobalConf) ParseSet(flagSetName string, set *flag.FlagSet) {
```

Comment:
I don't feel safe breaking the API without existing releases with semantic versioning in place, especially since I have code using globalconf in production. I think ParseSet is fine to keep, I was just a little uneasy about making a change to it without existing tests, so was looking to save the work of adding tests. Would you prefer I add tests proving it does what it says in the comment block? Now that I read that, It has a different behavior than what happens in Parse. It will overwrite already set flags.